ONSdigital/dp-frontend-renderer
# remove EnableCookiesControl flag

### What

- Removed `EnableCookiesControl` flag and references in templates as GDPR is now live
- Also removed `EnableJSONLDControl` flag and references in template now that GDPR is live
- Bumped `dp-frontend-models` to latest version which removes the `EnableJSONLDControl` value from the page model
- Bumped other ONS dependencies to their latest versions

### How to review

- Ensure relevant GDPR code is removed and that the cookies banner / preference page still load
- Ensure JSONLD scripts are still loaded in (homepage, CMD dataset/filter pages)
- Check this in tandem with [this PR for `dp-frontend-router`](https://github.com/ONSdigital/dp-frontend-router/pull/237)

### Who can review

Anyone
